pom.xml

```diff
@@ -58,7 +58,7 @@
     </scm>
 
     <properties>
-        <copyrightYear>2020</copyrightYear>
+        <copyrightYear>2021</copyrightYear>
         <copyrightNotice>Copyright ${copyrightYear} Micro Focus or one of its affiliates.</copyrightNotice>
```

Comment:
Drop this line.  It should only be there when the inception year and the copyright year are the same year.

Reply:
Done

Reply:
You've dropped the wrong line.

Reply:
You've dropped the wrong line.

Reply:
restored/dropped